CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Use [ruff](https://github.com/charliermarsh/ruff) instead of **isort** and **flake8** ([#1034](https://github.com/stac-utils/pystac/pull/1034))
 
+### Fixed
+
+- Item's `__geo_interface__` ([#1049](https://github.com/stac-utils/pystac/pull/1049))
```

Comment:
```suggestion
- Item `__geo_interface__` now correctly returns a Feature, rather than only the Geometry ([#1049](https://github.com/stac-utils/pystac/pull/1049))
```